ci/run_tests.sh

```diff
@@ -58,10 +58,10 @@ check_all_examples() {
     done
 }
 
-
-check_all_examples asmjs-unknown-emscripten
-check_all_examples wasm32-unknown-emscripten
+SHOWCASE=$(pwd)/examples/showcase
+check_example asmjs-unknown-emscripten $SHOWCASE
```

Comment:
This is possibly a separate issue, but I think you can use _separate_ travis instances for each "target".

This would help significantly reduce the amount of time one run takes as well.